ee/webhooks/cmd/fx-modules/fx-modules.go

```diff
@@ -0,0 +1,25 @@
+package fxmodules
+
+import (
+	"fmt"
+	"net/http"
+
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+
+	"go.opentelemetry.io/otel"
+)
+
+var Tracer = otel.Tracer("webhook")
+
+func FxProvideHttpClient() *http.Client {
+	return &http.Client{
+		Transport: otelhttp.NewTransport(http.DefaultTransport, otelhttp.WithSpanNameFormatter(func(operation string, r *http.Request) string {
+			str := fmt.Sprintf("%s %s", r.Method, r.URL.Path)
+			if len(r.URL.Query()) == 0 {
+				return str
+			}
+			return fmt.Sprintf("%s?%s", str, r.URL.Query().Encode())
```

Comment:
Same for query params. Worse then :D
You should use resource attributes to store those kind of data.

Reply:
This is code coming from V1.
I just used it as if because I Thought it was legacy.

Reply:
As i remember, i've done the same on membership, r.URL.Path return the templated string "/test/{id}". It can be easily checked with Jeager

Reply:
resolve